samples/highcharts/dragdrop/undraggable-points/demo.js

```diff
@@ -0,0 +1,62 @@
+Highcharts.chart('container', {
+    yAxis: {
+        softMin: -200,
+        softMax: 200
+    },
+
+    xAxis: {
+        min: 0.5,
+        max: 3.5
+    },
+
+    tooltip: {
+        enabled: false
+    },
+
+    title: {
+        text: 'Drag & drop disabled for specific points.'
+    },
+
+    series: [{
+        dragDrop: {
+            draggableY: true
+        },
+
+        point: {
+            events: {
+                drag: function () {
+                    if (this.x === 2) {
+                        return false;
+                    }
+                },
+                drop: function () {
+                    if (this.x === 2) {
+                        return false;
+                    }
+                }
```

Comment:
Checking for the `x` value it totally valid, but may not be practical. What if we want multiple points to be undraggable? What if the points also are `draggableX` so that x is not known?

I think it would be better and more extendable to mark the point with either a custom option or a `className`. 

```js
{
            x: 2,
            y: 1,
            color: '#f33',
            className: 'undraggable'
        }, 
```